db/seeds.rb

```diff
@@ -136,7 +134,7 @@
 sections.each do |section|
   puts "Creating students for section #{section.section_number}..."
   school = section.course.school
-  15.times do
+  10.times do
```

Comment:
on line 152, a fake student is being created inside this `10.times do` loop, nested inside of the `sections.each` loop... 
Is that line supposed to be inside the loop?